ribasim_qgis/pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools>=61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ribasim_qgis"
+description = "Ribasim QGIS Plugin"
+readme = "README.md"
```

Comment:
This file doesn't exist yet, but would be good to create, just like https://github.com/Deltares/Ribasim/blob/main/python/ribasim_api/README.md for instance.